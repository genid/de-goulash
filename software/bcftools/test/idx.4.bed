chr1	156134804	156134974
chr1	156135186	156135311
