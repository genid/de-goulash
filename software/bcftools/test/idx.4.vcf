##fileformat=VCFv4.2
##ALT=<ID=NON_REF,Description="Represents any possible alternative allele at this location">
##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
##contig=<ID=chr1,length=248956422,assembly=hg38>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	156134692	.	C	<NON_REF>	.	PASS	END=156135413
