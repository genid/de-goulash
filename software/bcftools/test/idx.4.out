chr1	156134692	.	C	<NON_REF>	.	PASS	END=156135413
